steps/google-drive-uploader/0.0.1/step.yml

```diff
@@ -0,0 +1,57 @@
+title: google-drive-uploader
```

Comment:
`title` should be the human-readable title of the step: https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#the-title.